apps/web/components/I18nLanguageHandler.tsx

```diff
@@ -1,16 +1,20 @@
+import parser from "accept-language-parser";
 import { useSession } from "next-auth/react";
 import { useTranslation } from "next-i18next";
 import { useEffect } from "react";
 
+import { locales } from "@calcom/lib/i18n";
 import { trpc } from "@calcom/trpc/react";
 
 // eslint-disable-next-line turbo/no-undeclared-env-vars
-const vercelCommitHash = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA || "NA";
+const CalComVersion = process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_SHA || "NA";
```

Comment:
@zomars i think we need to use the version from package.json here actually. 

If a self-hoster not running on Vercel runs this, it will only be able to call the i18n endpoint with the NA value, which means it will indefinitely cache what it gets back. Next time they upgrade, no new translations would be shown because they browser would still have the file cached

Reply:
Yeah.  I was trying to come up with a way to use the version without being too intrusive